app/views/account/projects/show.html.slim

```diff
@@ -3,21 +3,21 @@
     .level-left
       .level-item
         .title
-          | Project:
-          = @project.name
+          = "Project: #{@project.name}"
     .level-right
       = link_to "Add Task", new_account_project_task_path(@project), class: "button is-link"
   - @tasks.each do |task|
     .level
       .level-left
-       .level-item
-         = link_to task.title, account_project_task_path(@project, task)
-       .level-item
-         = link_to "Up", move_account_project_task_path(@project, task,
-           task: { row_order_position: :up}), method: :put, class: "button is-small left"
-       .level-item
-         = link_to "Down", move_account_project_task_path(@project, task,
-           task: { row_order_position: :down }), method: :put, class: "button is-small left"
+        .level-item
+          = link_to task.title, account_project_task_path(@project, task),
+            task.section? ? { class: 'title is-4 is-italic' } : nil
```

Comment:
`= link_to task.title, account_project_task_path(@project, task), { class: ('title is-4 is-italic' if task.section?) }`

Reply:
or = link_to task.title, account_project_task_path(@project, task), class: task_class(task)

helper:

def task_class(task)
  'title is-4 is-italic' if task.section?
end